rero_ils/modules/loans/api.py

```diff
@@ -470,20 +473,17 @@ def is_loan_overdue(self):
         return False
 
     def is_loan_due_soon(self, tstamp=None):
-        """Check if a loan is due soon."""
-        from .utils import get_circ_policy
-        if self.state != LoanState.ITEM_ON_LOAN:
-            return False
+        """Check if the loan is due soon.
 
-        circ_policy = get_circ_policy(self)
+        :param tstamp: a timestamp to define the execution time of the function.
+            Default to `datetime.now()`
+        :returns: True if is due soon
+        """
         date = tstamp or datetime.now(timezone.utc)
-        due_date = ciso8601.parse_datetime(self.end_date).replace(
-            tzinfo=timezone.utc)
-        days_before = circ_policy.due_soon_interval_days
-        if days_before:
-            start_date = ciso8601.parse_datetime(self.get('start_date'))
-            due_soon_date = due_date - timedelta(days=days_before)
-            return start_date < due_soon_date <= date < due_date
+        due_soon_date = self.get('due_soon_date')
+        """Check if a loan is due soon."""
+        if due_soon_date:
```

Comment:
Could you specify that this check if for legacy loan without 'due_soon_date' attribute ? 
Maybe a `TODO` comment to remember to remove this statement when legacy loan will be obsolete